pkg/scheduler/schedule_one_test.go

```diff
@@ -621,17 +621,17 @@ func TestSchedulerScheduleOne(t *testing.T) {
 			}
 			sched.scheduleOne(ctx)
 			<-called
-			if e, a := item.expectAssumedPod, gotAssumedPod; !reflect.DeepEqual(e, a) {
-				t.Errorf("assumed pod: wanted %v, got %v", e, a)
+			if got, want := gotAssumedPod, item.expectAssumedPod; !reflect.DeepEqual(got, want) {
```

Comment:
Sorry for missing the PR, one suggestion is can we use `cmp.Diff()` instead since we're modifying here, which is more recommended in kube-scheduler tests, you can refer to other tests for example. @mig-elgt 
https://github.com/kubernetes/kubernetes/blob/61d455ed1173cd89a98442adf4623a29c5681c58/cmd/kube-scheduler/app/server_test.go#L512-L514